lib/formatter/json-formatter.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = (genericFormatter) => {
+    return genericFormatter.formatWithFallback((req, res, body, done) => {
+        var string;
```

Comment:
Not sure I like the idea of using a variable named a type, maybe use `bufferContent`, or something else, just don't like `string`.